core/pkg/sync/grpc/grpc_sync.go

```diff
@@ -62,7 +63,16 @@ func (g *Sync) Init(_ context.Context) error {
 	}
 
 	// Derive reusable client connection
-	rpcCon, err := grpc.NewClient(g.URI, grpc.WithTransportCredentials(tCredentials))
+	// Set MaxMsgSize if passed
+    var rpcCon *grpc.ClientConn
+
+	if g.MaxMsgSize > 0 {
```

Comment:
QQ: should we add extra validation here and set Max and Min e.g. size should be > 4mb (default)  and < 20mb (max)

Reply:
I think validation here is fine. There could be users who wants to reduce the size for security, network reasons. Then they could use values less than 4MB